resources/js/components/Map/MapWidget.vue

```diff
@@ -0,0 +1,30 @@
+<script>
+export default {
+    name: 'MapWidget',
+    props: {
+        options: {
+            type: Object,
+            default: () => {}
+        },
+        filterValue: {
+            type: String,
+            default: ''
+        }
+    },
+    methods: {
+        selectionFilter (value) {
+            const reg = new RegExp(this.filterValue, 'gi');
+            return value.replace(reg, str => '<span class="ris-selection">' + str + '</span>');
+        }
+    }
+};
+</script>
+<template>
+    <a href="#" class="ris-map-widget">
```

Comment:
why do you need to use `<a>` tag here if there's no `href` and no any other event handlers on it?
if you click on it the page is gonna jump to the top, so i would suggest either put `@click.prevent` on it if it really needs to be `<a>` or use another tag